src/UI/Screens/EmailVerificationScreen.tsx

```diff
@@ -0,0 +1,43 @@
+import React, { useState } from 'react';
+import { View, Text, TextInput, TouchableOpacity, StyleSheet, SafeAreaView, Image } from 'react-native';
+import styles from './Email_Verification.styles'
+
+
+const EmailVerificationScreen = () => {
+  const [emailID, setEmailID] = useState('');
+  const [name, setName] = useState('');
+
+  return (
+    <View style={styles.container}>
+      <Image source={require('../../Assets/images/email_img.png')} style={styles.icon} />
+        <Text style={styles.title}>Select your personal</Text>
+        <Text style={styles.title}>email ID</Text>
+        <Text style={styles.subtitle} numberOfLines={2}>
+           This email ID will be linked {'\n'}
+          <Text style={styles.iconContainer}>to your account</Text>
+          </Text>
+        <TextInput
+          style={styles.input}
+          placeholder="Select your email ID"
+          placeholderTextColor={'#898A8D'}
```

Comment:
move this color to a colors.ts file and use from there

Reply:
Done Sir..